server/vcap-services.es6

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+// Environment variables
+
+const vcapConstants = {};
```

Comment:
I think this is a great idea, it could save everyone time. How should we go about initiating the conversation?